dissect/target/filesystems/config.py

```diff
@@ -85,18 +89,17 @@ def _get_till_file(self, path: str, relentry: FilesystemEntry) -> tuple[list[str
 
         return parts[idx:], entry
 
-    def get(
-        self, path: str, relentry: Optional[FilesystemEntry] = None, *args, **kwargs
-    ) -> Union[FilesystemEntry, ConfigurationEntry]:
-        """Retrieve a :class:`ConfigurationEntry` or :class:`.FilesystemEntry` relative to the root or ``relentry``.
+    def get(self, path: str, relentry: Optional[FilesystemEntry] = None, *args, **kwargs) -> ConfigurationEntry:
+        """Retrieve a :class:`ConfigurationEntry` relative to the root or ``relentry``.
 
         Raises:
             FileNotFoundError: if it could not find the entry.
         """
         parts, entry = self._get_till_file(path, relentry)
 
         if entry.is_dir():
-            return entry
+            # relative_path = fsutil.relpath(entry.path, self.root.top.path, alt_separator=self.alt_separator)
```

Comment:
please remove commented out code, unless it has a purpose (but then add a comment please)